vispy/gloo/texture.py

```diff
@@ -13,21 +13,19 @@
 from .util import check_enum
 
 
-F64_PRECISION_WARNING = ("GPUs can't support floating point data with more "
-                         "than 32-bits, precision will be lost due to "
-                         "downcasting to 32-bit float.")
+def downcast_to_32(data, copy=False):
```

Comment:
Can we include the idea of "if necessary" in the name here? Without knowledge of why this function exists (GPU's can't support >32 bits) reading its usage might cause something is always happening. I don't feel super strongly about this so if you disagree or it is too much work then fine.

Reply:
Done!